src/ansys/geometry/core/designer/component.py

```diff
@@ -484,6 +487,107 @@ def extrude_sketch(
         self._master_component.part.bodies.append(tb)
         return Body(response.id, response.name, self, tb)
 
+    @protect_grpc
+    @check_input_types
+    @ensure_design_is_active
+    def create_sweeping_profile(
```

Comment:
add  `@min_backend_version(24, 2, 0)` decorator to your function headers - this is new